backend/parkinglot/src/main/java/touk/demo/parkinglot/converter/SpotToOccupiedInfoConverter.java

```diff
@@ -0,0 +1,18 @@
+package touk.demo.parkinglot.converter;
+
+import org.springframework.stereotype.Component;
+import touk.demo.parkinglot.model.dto.OccupiedSpotInfo;
+import touk.demo.parkinglot.model.entity.Spot;
+
+@Component
+public class SpotToOccupiedInfoConverter implements BaseConverter<Spot, OccupiedSpotInfo> {
+
+  @Override
+  public OccupiedSpotInfo convert(Spot from) {
+    OccupiedSpotInfo info = new OccupiedSpotInfo();
```

Comment:
to co wcześniej - i dalej w klasach to samo

Reply:
+1